File: client/src/app/_models/reqIncMaxComicDto.ts
import { RequestIncMaxComicStatus } from '../_extensions/enumHelper';

export interface ReqIncMaxComicDto {
  id: number;
  email: string;
  quantity: number;
  request: string;
  creationTime: Date;
  status: RequestIncMaxComicStatus;
  processingDate: Date | null;
}

File: API/Dtos/ReqIncMaxComicDto.cs
using API.Helpers;

namespace API.Dtos
{
    public class ReqIncMaxComicDto
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public int Quantity { get; set; }
        public string Request { get; set; }
        public DateTime CreationTime { get; set; }
        public RequestIncMaxComicStatus Status { get; set; }
        public DateTime? ProcessingDate { get; set; }
    }
}